api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -316,6 +316,12 @@ message Watchdog {
   // kill behavior. If not specified the default is 0 (disabled).
   google.protobuf.Duration kill_timeout = 3;
 
+  // Defines the maximum skew used to adjust the *kill_timeout* if *kill_timeout* is
+  // enabled. Enabling this feature would help to reduce risk of synchronized
+  // watchdog kill events across proxies due to external triggers. Set to 0 to
+  // disable. If not specified the default is 0 (disabled).
+  google.protobuf.Duration max_kill_timeout_jitter = 6;
```

Comment:
Maybe add a pgv validation to ensure that this is gte 0?

looks something like this
```
  google.protobuf.Duration session_timeout = 6 [(validate.rules).duration = {
    lt {seconds: 4294967296}
    gte {}
  }];
```

Reply:
Good idea, done.